app/controllers/concerns/appropriate_body_selection/controller.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module AppropriateBodySelection
+  module Controller
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :load_appropriate_body_form
+
+      helper_method :appropriate_body_from_path, :appropriate_body_school_name
+    end
+
+    def appropriate_body_appointed
+      render "/appropriate_body_selection/body_appointed"
+    end
+
+    def update_appropriate_body_appointed
+      if @appropriate_body_form.valid? :body_appointed
+        store_appropriate_body_form
+        if @appropriate_body_form.body_appointed?
+          redirect_to action: :appropriate_body_type
+        else
+          method(appropriate_body_submit_action).call
```

Comment:
Just wondering what's the difference between doing the method().call and just calling the method `appropriate_body_submit_action`

Reply:
So is that to stop data from a different journey using data in another session? and then causing problems with what's submitted?